### PR TITLE
Fix removing styles from function rules 

### PR DESCRIPTION
**Pull request by kof**
#1026

### CODE REVIEW

File: packages/jss/tests/functional/sheet.js [hunk 1]
```diff
@@ -515,6 +515,22 @@ describe('Functional: sheet', () => {
     })
   })
 
+  describe('sheet.update()', () => {
```
Comment (HenriBeck) [inline packages/jss/tests/functional/sheet.js]: Please also add a test where the property becomes `null` and `false`

Comment (kof) [reply packages/jss/tests/functional/sheet.js]: will do

**Comment by HenriBeck** [comment]
A changelog is missing as well.

**Comment by kof** [comment]
> A changelog is missing as well.

Forgot to push, fixed
